src/compose/ComposeBox.js

```diff
@@ -45,7 +45,6 @@ import MentionWarnings from './MentionWarnings';
 import {
   getAuth,
   getIsAdmin,
-  getLastMessageTopic,
```

Comment:
```
We should follow up by having the "Topic" input be the default thing
that's focused, rather than the "Message" input.
```

This seems like it'd be pretty easy to do, with the `autoFocusTopic` prop you introduce later in the PR? 😅

Reply:
Sadly not — this is referring to when you go and tap on the "Message" input box, it should actually focus the topic input box, but we don't want to `autoFocus` it in the sense that it'll grab your keyboard as soon as you load the message.